ResearchKit/Printing/Base.lproj/HTMLTemplates.strings

```diff
@@ -0,0 +1,45 @@
+/*
+ Copyright (c) 2016, Oliver Schaefer
+ 
+ Redistribution and use in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ 
+ 1.  Redistributions of source code must retain the above copyright notice, this
+ list of conditions and the following disclaimer.
+ 
+ 2.  Redistributions in binary form must reproduce the above copyright notice,
+ this list of conditions and the following disclaimer in the documentation and/or
+ other materials provided with the distribution.
+ 
+ 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+ may be used to endorse or promote products derived from this software without
+ specific prior written permission. No license is granted to the trademarks of
+ the copyright holders even if such marks are included in this software.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+"HTML" = "<!doctype html><html><head><title>html</title><meta charset=\"utf-8\"></head><style>%@</style><body>%@</body></html>";
+"STEP" = "<div class=\"stepSeparator\">%@%@%@</div>";
+/* STEP HEADER */
+"STEP_HEADER" = "<p id=\"stepTitle\">%@</p><p id=\"stepText\">%@</p>";
+/* STEP BODY */
+"FORM_STEP" = "<p id=\"formStepTitle\">%@</p>";
+"FORM_STEP_ANSWER" = "<p id=\"sectionTitle\">%@</p><table id=\"answerTable\">%@</table>";
+"QUESTION_STEP_ANSWER" = "<table id=\"answerTable\">%@</table>";
+"STEP_ANSWER" = "<tr class=\"answerRow\"><td><div class=\"answerColumn\"/>%@<td/></tr>";
+"STEP_SELECTED_ANSWER" = "<tr class=\"selectedAnswerRow\"><td><div class=\"selectedAnswerPrimaryColumn\"/>%@<td/><td><div class=\"selectedAnswerSecondaryColumn\">%@</div></td></tr>";
+"STEP_UNSELECTED_ANSWER" = "<tr class=\"unselectedAnswerRow\"><td><div class=\"unselectedAnswerPrimaryColumn\"/>%@<td/><td><div class=\"unselectedAnswerSecondaryColumn\">%@</div></td></tr>";
+/* STEP FOOTER */
+"STEP_FOOTER" = "<p class=\"stepFooter\">%@</p>";
+/* IMAGE */
+"IMAGE" = "<figure class=\"figure\"><img class=\"image\" height=\"%@\" width=\"%@\" src=\"data:image/png;base64,%@\"/><figcaption>%@</figcaption></figure>";
```

Comment:
This is not about localization. I wanted to separate the html format strings from the code itself. That way the strings are easy to maintain and all live in the same place. If anybody wants to modify or add strings later on, it will be a lot clearer. If you know a better approach than a strings file to achieve this, let me know.